### PR TITLE
Web3api (Raymond.frontier up 3.2.0)

### PR DESCRIPTION
**Pull request by alanzhong**
@raykyri  put up the web3api patch (https://github.com/hicommonwealth/edgeware-node/issues/294)

### CODE REVIEW

**Comment by raykyri** [comment]
@drewstone how are we using the `master` branch, can we merge this? This PR adds a missing API to the node client (not the runtime).

File: Cargo.toml [hunk 1]
```diff
@@ -71,17 +71,17 @@ zeroize = { opt-level = 3 }
...
-fp-consensus = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fp-evm = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-fp-rpc = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-ethereum = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-ed25519 = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-modexp = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
-pallet-evm-precompile-simple = { git = "https://github.com/hicommonwealth/frontier.git", rev = "d3179acb5a111d58af6c48a12588ecd38750a9bb" }
+fc-consensus = { git = "https://github.com/VLT-Protocol/frontier.git", rev = "6a284edd6bf34e96d1a4721fad215a0adb2cd5d3" }
```
Comment (drewstone) [inline Cargo.toml:74]: This fork is potentially an issue. Have you tested this against a live chain? Connected to beresheet/mainnet using this new build?

Have you synced from scratch on both beresheet/mainnet?

**Review by drewstone — CHANGES_REQUESTED** [review]
Need to learn more about this fork, can you link what's different and what patches have been made @raykyri @alanzhong ?

File: Cargo.toml (hunk 1 above)
Comment (drewstone) [reply Cargo.toml:74]: There are fixes in our fork that make it possible to process the initial runtime upgrade that may be missing in this fork. Without testing a full sync, I do not feel comfortable merging this in. Please create cloud instances and try to sync and circle back.

Comment (alanzhong) [reply Cargo.toml:74]: @raykyri did you have a fork from VLT when you made the patch

Comment (raykyri) [reply Cargo.toml:74]: This is the only change in the VLT branch:
https://github.com/VLT-Protocol/frontier/commit/6a284edd6bf34e96d1a4721fad215a0adb2cd5d3

**Pull request closed without merging**